### ISSUE TITLE
"a" placeholder link on mode bar buttons doesn't work on some screen readers

### ISSUE THREAD

**Issue by ghost** [plotly/plotly.js#6941]
We are using a tool called Sort Site to evaluate our site for Section 508 compliance.  It is reporting the following error:
![image](https://github.com/plotly/plotly.js/assets/86489380/d627a448-7802-48a7-8c44-5aff7c519e24)

This is being reported for every button on the mode bar.  Perhaps replace the "a" with a different element?

**Comment by emilykl** [comment]
Modebar buttons were changed from `<a>` elements to `<button>` elements in https://github.com/plotly/plotly.js/pull/7492, first released in plotly.js v3.1.0.
